### DISCUSSION TITLE
Windows XP Shared Directory: Broken???

### DISCUSSION THREAD

**Discussion by DevBlue7** [utmapp/UTM#6579] [Windows]
Shared Directory doesn't work on windows XP. I'm using a Windows XP prebuilt from the gallery, is there any way to fix this? I can't find the folder anywhere. Some people said it's broken, and one of the methods are making file servers or bundling folders as hybrid .iso images, but I don't have a computer.

I heard it's broken, but I want to confirm whether it's true or not

**Configuration (required)*
* UTM Version: 
UTM HV, 4.5.3
* OS Version: 
15.0.2
* Device Model: 
iPad 8th Gen
* Is it jailbroken (name jailbreak used)? 
No
* How did you install UTM? 
TrollStore

**Comment by compfinal** [comment]
Same issue here using UTM SE

**Comment by osy** [comment]
You need to install an older spice-webdavd: https://www.spice-space.org/download/windows/spice-webdavd/spice-webdavd-x86-2.2.msi

**Reply by compfinal** [reply]
Thank you!

**Reply by osy** [reply]
Actually, you only need spice-webdavd, updated the link https://www.spice-space.org/download/windows/spice-webdavd/spice-webdavd-x86-2.2.msi
